http-proxy/conf.d/ngnix.conf

```diff
@@ -2,6 +2,8 @@
 server {
 
     listen 80;
+    client_max_body_size 8M;
```

Comment:
I'd like a bit of background to know why this is needed. We are downloading large files from the server, but when are we uploading large files?